player.cpp

```diff
@@ -0,0 +1,31 @@
+#include "player.h"
+
+Player::Player()
+{
+
+}
+
+void Player::intitialize(Ui::MainWindow *ui_ptr)
+{
+    ui = ui_ptr;
```

Comment:
if we add another ui into our game then this will be a problem (if we forget to re-initialize)

Reply:
okay, me gotta tell meself that me understand that